src/visual/Visualizer.js

```diff
@@ -86,6 +86,20 @@ export default class Visualizer {
     this._timer = null;
     this._step = null;
     this.clear();
+
+    const paperDefaultValidateConnectionMethod = this.paper.options.validateConnection;
```

Comment:
Won't `oldValidateConnection` or just `validateConnection` be shorter and more clear?